docs/quickstart.md

```diff
@@ -58,7 +58,9 @@ export SPARK_CONF_DIR=$SPARK_HOME/conf
 export PATH=$JAVA_HOME/bin:$HIVE_HOME/bin:$HADOOP_HOME/bin:$SPARK_INSTALL/bin:$PATH
 ```
 
-### DataSource API
+### Two different API's
```

Comment:
Unless there is some explanation about which API to use when (which may be good to add), can we remove the "Two different APIs" heading/.. it just nests the doc more without clear value

Reply:
added some explanation, I think this heading gives a structure to 2 subheadings, otherwise 2 subheadings without an umbrella looked weird when I was reading the whole document.